.changeset/perfect-wasps-grow.md

```diff
@@ -0,0 +1,20 @@
+---
+'astro': patch
+---
+
+Exposes `z` from the new `astro:schema` module. This is the new recommended import source for all Zod utilities in Astro. `z` will still be available to import from existing Astro modules (`astro:content` and `astro:actions`), but this will change in future releases:
+
+- `z` will be **removed** from `astro:actions` in the next minor release.
+- `z` will be **deprecated** in `astro:content` in the next major release.
```

Comment:
Shouldn't we add a JSDoc `deprecated` for `z` exposed from `astro:content`?

Reply:
Why not? Seems reasonable to me at first glance. Explain the reason for the change, and that's part of it.